code-of-conduct-faq.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Code of Conduct FAQ
+slug: code-of-conduct-faq
+---
+
+# Code of Conduct FAQ
```

Comment:
This error can be ignored. It doesn't take in account for the Jekyll meta data 
![Image 2022-03-22 at 8 09 10 AM](https://user-images.githubusercontent.com/398230/159514363-2bc419f2-c5d1-45dd-886f-98ffcfb3d791.jpg)